cocos/rendering/define.ts

```diff
@@ -1121,3 +1121,21 @@ export function isEnableEffect (): boolean {
 }
 
 /* eslint-enable max-len */
+
+export function getPassPool (): RecyclePool<IRenderPass> {
+    return new RecyclePool<IRenderPass>((): {
```

Comment:
Do we need to return `new RecyclePool` each time invoking `getPassPool`?
If so, perhaps `createPassPool` is a better name.

Reply:
This will not be called every frame, but only once at initialization.